src/FrontendModuler/Textarea.elm

```diff
@@ -119,7 +119,7 @@ toHtml (Textarea options) =
             Just feilmelding ->
                 div [ role "alert", ariaLive "assertive" ]
                     [ div [ class "skjemaelement__feilmelding" ]
-                        [ text feilmelding ]
+                        [ p [ class "typo-feilmelding" ] [ text feilmelding ] ]
```

Comment:
styling was wrong on the error message, this should make it the same as other error message styles